Fix SystemTray.Update() if tooltip was initially None

elibroftw opened the pull request:
- Removed if self.icon because the statement will always return True.
- self.Tooltip should be be set either as tooltip or empty string, otherwise TaskBarIcon.SetIcon() will fail sometimes since self.Tooltip will be None.

My thoughts: tooltip default should be ''; an empty string but since that is a design change, I didn't do it.

Also, I think you would benefit from using [ImgBot](https://github.com/apps/imgbot).
Here are the [changes](https://github.com/elibroftw/PySimpleGUI/pull/1#issue-298723300) it made.
Refer to #1690 specifically [this](https://github.com/PySimpleGUI/PySimpleGUI/issues/1690#issuecomment-512615563).

MikeTheWatchGuy commented:
I've been looking through your changes.  

Did you read the pinned instructions on how to do a Pull Request on this project.

One problem with simply going off and making a bunch of changes and submitting them is that you have no clue what my local copy of the code looks.  I don't always check in my code right away, because, I'm the only one working on the code.  

I had already made some of the changes and I'm unsure of the others so it's going to take me a little bit.

Let's take a quick look at one set of changes so you can see what I mean:
![image](https://user-images.githubusercontent.com/13696193/61499263-99142900-a994-11e9-8923-375a359f94da.png)

I had already removed the `or wx.EmptyString` that I think you suggested removing in your text, but inexplicably still shows up in your code.

I'm going to close this request as a result, and will check in my code as soon as I can so you can see if the problem persists.

Modifying the core code is something that requires coordinating as you can see.  Give me a heads up (and some time) and I'll get back to you on where things are.

elibroftw commented:
No sorry I did not read any pinned instructions because I could not see any and did not encounter any. I had added the or statement in because otherwise the code would bug out if I updated the tray without providing a tooltip. I suggested making the parameter tooltip=wx.EmptyString but since most of your parameters default as None I decided not to do that and to only fix the bug. To see the bug yourself, you can create a tray application without a tooltip and then try to update the tray without supplying a new tooltip. You'll encounter an error. I'm okay with not accept my pull request but at least just see the changes when you working on it.

elibroftw commented:
And you also did not respond to the IMG bot thing.

MikeTheWatchGuy commented:
I didn't respond to your IMG bot thing yet.  I've done nothing but address people's issues and answer questions since 2 AM this morning.  It's 9:33 PM at the moment and I got complete zero of the planned items I have for the day.
The response to IMG bot is that I'm not in need of it.  Pull requests are formally shutting down.  I'll post more about it soon.  It's been in the works.

>  at least just see the changes when you working on it.

Did I not see the screen shot  of your code on my screen in a side by side diff in my response?
